### PR TITLE
Impr/1.6 network testcases

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Added tcs for - 
1) PBRs
2) Floating IPs
3) Static Routes

Added datasource for pbrs .

### CODE REVIEW

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.8) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.8) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.8) is below threshold (50). Kindly add more acceptance tests.

File: nutanix/data_source_nutanix_floating_ip_test.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFloatingIPDataSourceConfig(r),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.nutanix_floating_ip.test", "spec.0.resources.0.external_subnet_reference.kind", "subnet"),
+					resource.TestCheckResourceAttr("data.nutanix_floating_ip.test", "spec.0.resources.0.vpc_reference.kind", "vpc"),
+				),
```
Comment (bhati-pradeep) [inline nutanix/data_source_nutanix_floating_ip_test.go:22]: check more details if possible which are unique for all entities like names, etc. Also some IPs, just to confirm we are not omitting any Set.

File: nutanix/data_source_nutanix_floating_ips_test.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFloatingIPsDataSourceConfig(r),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.nutanix_floating_ips.test", "entities.#"),
+					resource.TestCheckResourceAttrSet("data.nutanix_floating_ips.test", "api_version"),
+					resource.TestCheckResourceAttr("data.nutanix_floating_ips.test", "entities.0.spec.0.resources.0.external_subnet_reference.kind", "subnet"),
```
Comment (bhati-pradeep) [inline nutanix/data_source_nutanix_floating_ips_test.go]: Same, please add more checks if possible

File: nutanix/data_source_nutanix_static_routes.go [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+			"spec": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"resources": {
```
Comment (bhati-pradeep) [inline nutanix/data_source_nutanix_static_routes.go]: No schema for default static route ? It has separate fields for it

**Comment by bhati-pradeep** [comment]
Also please add fields for local static routes and default static routes in data source.
Add default routes in resource as well.

**Comment by bhati-pradeep** [comment]
Add fields for all kind of static routes in data source

File: nutanix/data_source_nutanix_static_routes.go (hunk 3 above)
Comment (Abhishekism9450) [reply nutanix/data_source_nutanix_static_routes.go]: created the schema for default , local, dynamic route.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (41.9) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (42.5) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.7) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.9) is below threshold (50). Kindly add more acceptance tests.

**Review by siddharth-kulshrestha — APPROVED** [review]

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.9) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (43.6) is below threshold (50). Kindly add more acceptance tests.

**Pull request merged**